spec/features/user_views_upcoming_events_spec.rb

```diff
@@ -9,16 +9,22 @@
       expect(page).to have_content("thoughtbot")
       expect(page).to have_content("41 Winter Street, 6th Floor")
       expect(page).to have_content("Boston")
-      expect(page).to have_content("6:00PM to 8:00PM")
     end
   end
 
   def stub_request_to_meetup_api
     stub_request(:get, /api.meetup.com/).
-      to_return(status: 200, body: events_json)
+      to_return(status: 200, body: [future_meetup_event].to_json )
   end
 
-  def events_json
-    File.read(Rails.root.join("spec/fixtures/meetup_events.json"))
+  def future_meetup_event
+    future_time_ms = 2.days.from_now.to_i * 1000
+    event_data.merge("time" => future_time_ms)
+  end
+
+  def event_data
+    JSON.parse(
+      File.read(Rails.root.join("spec/fixtures/meetup_event.json"))
```

Comment:
It might be outside the scope of this PR, but [`ActiveSupport::Testing::FileFixtures` defines a `#file_fixture` helper](https://api.rubyonrails.org/classes/ActiveSupport/Testing/FileFixtures.html#method-i-file_fixture) to abstract over fixture file fetching.

Reply:
thanks! 🌟